portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+// import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class MyServlet extends HttpServlet {
+
+  String[] info = {"Hi", "This is about", "My information"};
+
+  String[] list;
+
+  public MyServlet(String[] list){
+      this.list = list;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Convert the server info to JSON
+    MyServlet myservlet = new MyServlet(info);
+    String json = convertToJson(myservlet);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(MyServlet myservlet) {
+    String json = "{";
+    json += "\"first\": ";
+    json += "\"" + myservlet.list[0] + "\"";
+    json += ", ";
+    json += "\"second\": ";
+    json += "\"" + myservlet.list[1] + "\"";
+    json += ", ";
+    json += "\"third\": ";
+    json += "\"" + myservlet.list[2] + "\"";
+    json += "}";
+    return json;
```

Comment:
Couple notes. If you want to create a long multi line string you can do the concatenation with one statement, like 

```
// not correct JSON, but you get the picture
String json = "{";
 + "key: value"
 + "anotherKey: anotherValue";

```

But we also don't want to hard code this JSON conversion because a) it's difficult to do correctly (as demonstrated with this exercise) and b) if you add another info string you have to update this hard coded conversion. Whereas using an `ArrayList` and `Gson` will scale automatically - all you have to do is add a new string to the list.


Note: Updating this means updating how you handle the json object in JS code.

Reply:
I don't know which step I was doing wrong, but when I tried "import com.google.gson.Gson; ", it gave me error. And this is why I commented out all things about Gson. I tried to import in MyServlet.java file, is this a correct place to import? And is there anything else I need to do? Thank you.